webapp/src/manifest.ts

```diff
@@ -107,6 +99,14 @@ const manifest = JSON.parse(`
                 "help_text": "(Optional) The number of minutes from when the meeting link is created to when it becomes invalid. Minimum is 1 minute. Only applies if using JWT authentication for your Jitsi server.",
                 "placeholder": "",
                 "default": 30
+            },
```

Comment:
This comes from other PR, but it is ok to go here.

Reply:
Thanks :+1: